src/app/PCM/templates/tpl_horizontal_cards/index.js

```diff
@@ -86,7 +86,7 @@ const Card = ({ mappedValues, baseEntityCode, actions, sbeCode, primaryColor })
               <Attribute
                 code={baseEntityCode}
                 attribute={value}
-                config={{ carddisplay: 'true', showSingleImgOnly: 'true' }}
+                config={{ carddisplay: 'true', showsingleimgonly: 'true' }}
```

Comment:
can we ask the backend to send a boolean value instead of having true and false as a string please.

Reply:
BE has certain validation. If extra identifiers are sent as attributes, it gives error in BE. So, this has been done in FE. However, Trimik is going to have a look at it later.